python/pipx/Portfile

```diff
@@ -20,7 +20,21 @@ checksums           rmd160  64b198d937c3c7b12531a81828c7b813fa9a597a \
 
 python.default_version 37
 python.versions     36 37 38
-depends_lib         port:py${python.version}-argcomplete \
+
+depends_lib         port:py${python.version}-userpath \
```

Comment:
please change this to ```depends_lib-append``` as not to overwrite the dependencies added by the python PG and keep the dependencies sorted alphabetically.

Secondly,  *I think* that the dependency listing has to come after you've set the python version using your variants to make sure that it knows which version to use.

Reply:
Although that was not your mistake but mine, let's take the opportunity to change to `depends_lib-append`.

And yes, please keep the ports sorted alphabetically.